tests/ci/avalon-multiple-wp-direct-model-tests.yaml

```diff
@@ -0,0 +1,154 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  avalon-multiple-wp-direct-model-tests:
+    image: avalon-shell:${ISOLATION_ID}
+    working_dir: "/project/avalon/"
+    volumes:
+      - ../logs:/project/avalon/logs
+    entrypoint: "bash -c \"\
+        ./tools/run_tests.sh -l avalon-listener \""
+    depends_on:
+      - avalon-enclave-manager
+      - avalon-listener
+
+  avalon-kme:
+    container_name: avalon-kme
+    image: avalon-enclave-manager-kme:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 1948
+      - 5555
+    command: |
+      bash -c "
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --bind http://avalon-kme:1948
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+      - avalon-listener
+
+  avalon-enclave-manager:
+    container_name: avalon-wpe
+    image: avalon-enclave-manager-wpe:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 5555
+    command: |
+      bash -c "
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --kme_listener_url http://avalon-kme:1948
+        tail -f /dev/null
+      "
+    depends_on:
+      - avalon-lmdb
+      - avalon-listener
+      - avalon-kme
+
+  avalon-kme-2:
+    container_name: avalon-kme-2
+    image: avalon-enclave-manager-kme:${ISOLATION_ID}
+    environment:
+      - http_proxy
+      - https_proxy
+      - no_proxy
+    expose:
+      - 1949
+      - 5555
+    command: |
+      bash -c "
+        enclave_manager --lmdb_url http://avalon-lmdb:9090 --bind http://avalon-kme-2:1949 --worker_id worker_pool2
```

Comment:
No need to change port from 1948 to 1949 for avalon-kme-2. You can use the same port 1948.